Paws4Me/Paws4Me/Utilities/URLSession+Extension.swift

```diff
@@ -14,14 +14,14 @@ extension URLSession {
         case invalidUrl
         case invalidData
     }
-    func makeRequest<Generic: Codable>(url: URLRequest?,
+    func makeRequest<Generic: Codable>(url: URL?,
                                        model: Generic.Type,
                                        completion: @escaping (Result<Generic, Error>) -> Void) {
         guard let endpointUrl = url else {
             completion(.failure(CustomError.invalidUrl))
             return
         }
-        let apiTask = self.dataTask(with: endpointUrl as URLRequest) { data, _, error in
+        let apiTask = self.dataTask(with: endpointUrl as URL) { data, _, error in
```

Comment:
Also no need to cast here 😄

`let apiTask = self.dataTask(with: endpointUrl) { data, _, error in`

Reply:
I agree, thank you very much